CHANGELOG.md

```diff
@@ -19,6 +19,61 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## lb_clmm [0.8.2] - PR #115
+
+### Added
+
+- Add a new endpoint `initialize_customizable_permission_lb_pair`, that allows pool creator to be able to create pool with input `active_id`, `bin_step`, `base_factor`, `activation_point` and `alpha-vault`
```

Comment:
The new endpoint is `initialize_customizable_permissionless_lb_pair`

Reply:
Fixed